examples/VLCB_1in1out/VLCB_1in1out.ino

```diff
@@ -205,17 +208,9 @@ void processModuleSwitchChange()
 {
   if (moduleSwitch.stateChanged())
   {
-    byte opc = moduleSwitch.isPressed() ? OPC_ACON : OPC_ACOF;
+    bool state = moduleSwitch.isPressed() ? true : false;
```

Comment:
You can remove the "? true : false" part as .isPressed() returns the boolean you want to set state to.

Reply:
Of course it does!  Silly me!